windows/window_plus_plugin.cpp

```diff
@@ -694,6 +703,15 @@ void WindowPlusPlugin::HandleMethodCall(
     // Request focus.
     ::SetForegroundWindow(GetWindow());
     result->Success();
+  } else if (method_call.method_name().compare(kSetMinimumSizeMethodName) == 0) {
+    auto args = std::get<flutter::EncodableMap>(*method_call.arguments());
+    SetMinimumSize(args);
+    result->Success();
+  } else if (method_call.method_name().compare(kGetMinimumSizeMethodName) == 0) {
+    auto sizeMap = std::map<flutter::EncodableValue, flutter::EncodableValue>();
```

Comment:
We follow Google C++ style guide. Snake case is used for local variable names.

https://google.github.io/styleguide/cppguide.html#Variable_Names